packages/now-build-utils/src/prerender.ts

```diff
@@ -32,6 +40,22 @@ export class Prerender {
     }
     this.group = group;
 
+    if (bypassToken == null) {
+      this.bypassToken = null;
+    } else if (typeof bypassToken === 'string') {
+      if (bypassToken.length < 32) {
+        // Enforce 128 bits of entropy for safety reasons (UUIDv4 size)
+        throw new Error(
+          'The `bypassToken` argument for `Prerender` must be 32 characters or more.'
+        );
+      }
+      this.bypassToken = bypassToken;
+    } else {
+      throw new Error(
+        'The `bypassToken` argument for `Prerender` must be a `string`.'
+      );
```

Comment:
You're comment on line 11 says that its optional and can therefore be undefined. So why throw here when undefined?

Reply:
Undefined and null are already handled by this line. This catches non-string values (number, boolean).